test/unit/component/mxgraph/config/StyleConfigurator.test.ts

```diff
@@ -276,9 +276,11 @@ describe('mxgraph renderer', () => {
     //[ShapeBpmnElementKind.AD_HOC_SUB_PROCESS],
     //[ShapeBpmnElementKind.TRANSACTION],
   ])('compute style - markers for %s', (bpmnKind: ShapeBpmnElementKind) => {
-    it(`${bpmnKind} with Loop marker`, () => {
-      const shape = newShape(newShapeBpmnElement(bpmnKind, ShapeBpmnMarkerKind.LOOP), newLabel({ name: 'Arial' }));
-      expect(computeStyle(shape)).toEqual(`${bpmnKind};bpmn.markers=loop;fontFamily=Arial`);
+    describe.each([[ShapeBpmnMarkerKind.LOOP], [ShapeBpmnMarkerKind.MULTI_INSTANCE_SEQUENTIAL]])(`compute style - %s marker for ${bpmnKind}`, (markerKind: ShapeBpmnMarkerKind) => {
```

Comment:
:warning: this change requires to update the dedicated test for subprocess (expand + other marker) which should be parametrized as well.
currently, it is run twice with the loop marker instead of once with the loop,  once with the multi instance
Otherwise, we can put this specific test out of this block because we may not want to test expand marker + every other marker

Reply:
I was wrong the subprocess test is out of the describe block, my bad, sorry.